### PR TITLE
Feature: Bybit + Customized account ID

### PR DESCRIPTION
**Pull request by MinsungKim-BlockCrafters**
## What does this PR do?

Add Bybit and customized account ID.

## Changes

- Added `self.custom_account_id` on `Feed.__init__`
- Implemented Bybit private data parsing

### CODE REVIEW

File: cryptofeed/exchanges/bitmex.py
Comment (boseok-bc) [inline cryptofeed/exchanges/bitmex.py]:
```suggestion
        if self.custom_account_id or data.get('account') is not None:
```

File: cryptofeed/exchanges/bitmex.py
Comment (boseok-bc) [inline cryptofeed/exchanges/bitmex.py]:
```suggestion
        if self.custom_account_id or data.get('account') is not None:
```

File: cryptofeed/exchanges/bybit.py
Comment (boseok-bc) [inline cryptofeed/exchanges/bybit.py]:
```suggestion
        elif "position" == msg["topic"]:
```
I'm not sure this was tested. Could you check whether position case was tested?

File: cryptofeed/exchanges/bybit.py
Comment (boseok-bc) [inline cryptofeed/exchanges/bybit.py]: I think existence of `stop_order` is the reason why it use `in` statement.
https://bybit-exchange.github.io/docs/inverse/#t-websocketstoporder

File: cryptofeed/exchanges/bybit.py
Comment (boseok-bc) [inline cryptofeed/exchanges/bybit.py]: Could you unify both exchanges to use `average`?

File: cryptofeed/exchanges/bybit.py
Comment (boseok-bc) [inline cryptofeed/exchanges/bybit.py]: They are same, though, Let's use `cum_exec_qty` 
https://bybit-exchange.github.io/docs/inverse/#t-websocketorder

File: cryptofeed/exchanges/bybit.py
Comment (boseok-bc) [inline cryptofeed/exchanges/bybit.py]:
```suggestion
        if self.custom_account_id or data.get('user_id') is not None:
```

File: cryptofeed/exchanges/bybit.py
Comment (boseok-bc) [inline cryptofeed/exchanges/bybit.py]: Shouldn't it be `currency`? There's no symbol on response of `parse_balance`.

**Review by boseok-bc — COMMENTED** [review]
You may wanna test some channels with real connection.

File: cryptofeed/exchanges/bybit.py
Comment (MinsungKim-BlockCrafters) [reply cryptofeed/exchanges/bybit.py]: I didn't think that I should fetch `stop_order`, thanks!

File: cryptofeed/exchanges/bybit.py
Comment (MinsungKim-BlockCrafters) [reply cryptofeed/exchanges/bybit.py]: Yes, it wasn't tested.. Sorry for such mistake. I planned like I have to test with actual running with nile because there was no such testing code for Bitmex in cryptofeed repository..

File: cryptofeed/exchanges/bybit.py
Comment (MinsungKim-BlockCrafters) [reply cryptofeed/exchanges/bybit.py]: Ok. For reference, look at here: https://docs.ccxt.com/en/latest/manual.html#order-structure

File: cryptofeed/exchanges/bitmex.py
Comment (MinsungKim-BlockCrafters) [reply cryptofeed/exchanges/bitmex.py]: Fixed.

File: cryptofeed/exchanges/bitmex.py
Comment (MinsungKim-BlockCrafters) [reply cryptofeed/exchanges/bitmex.py]: Fixed.

File: cryptofeed/exchanges/bybit.py
Comment (MinsungKim-BlockCrafters) [reply cryptofeed/exchanges/bybit.py]: Fixed.

File: cryptofeed/exchanges/bybit.py
Comment (MinsungKim-BlockCrafters) [reply cryptofeed/exchanges/bybit.py]: Fixed.

File: cryptofeed/exchanges/bybit.py
Comment (MinsungKim-BlockCrafters) [reply cryptofeed/exchanges/bybit.py]: You are right. Fixed.

File: cryptofeed/exchanges/bybit.py
Comment (boseok-bc) [reply cryptofeed/exchanges/bybit.py]: Okay as we talked at slack, Let's use demo script.

File: cryptofeed/exchanges/bybit.py
Comment (boseok-bc) [reply cryptofeed/exchanges/bybit.py]: The document is well organized. 👍

**Review by boseok-bc — APPROVED** [review]

**Comment by MinsungKim-BlockCrafters** [comment]
[WARNING] Do NOT MERGE until PR #62 is merged!

**Review by KunduzB — APPROVED** [review]

**Pull request merged**
